etc/profile-a-l/irssi.profile

```diff
@@ -0,0 +1,67 @@
+# Firejail profile for irssi
+# Description: IRC client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include irssi.local
+
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.irssi/
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-X11.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.irssi/
+whitelist ${HOME}/.irssi/
+
+#include whitelist-usr-share-common.inc
+#include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+no3d
+nodvd
+nogroups
+nonewprivs
+noprinters
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+# NOTE: comments here are things that can be improved, if you can spare the time.
+##seccomp.drop SYSCALLS (see syscalls.txt)
+##seccomp-error-action log (only for debugging seccomp issues)
+#shell none
+#tracelog
+disable-mnt
+##private-opt NAME
+#private-tmp
```

Comment:
`private-dev`? `private-cache`?

Reply:
lost those when moving my profile to the template. thanks.